README.md

```diff
@@ -236,6 +253,57 @@ open-attestation document-store revoke --network ropsten --address 0x19f89607b52
 ✔  success   Document/Document Batch with hash 0x0c1a666aa55d17d26412bb57fbed96f40ec5a08e2f995a108faf45429ae3511f has been revoked on 0x19f89607b52268D0A19543e48F790c65750869c6
 ```
 
+## Paymaster
+
+### set-target
+
+Registers a contract with a paymaster deployed on the blockchain is willing to pay for
+
+```bash
+open-attestation paymaster set-target --targetAddress <GSN_CAPABLE_DOCUMENT_STORE_ADDRESS> --paymasterAddress <PAYMASTER_ADDRESS> [options]
```

Comment:
let's use kebab-case:
- targetAddress => target-address
- paymasterAddress => paymaster-address

It should work out of the box already, just need to change the doc for consistency